collector.go

```diff
@@ -185,10 +196,11 @@ func (c Collector) measure() error {
 		var measurementsPrecision []float64
 		var measurementsLeap []float64
 
-		log.Printf("WARN: clock drift is above %.2fs, taking multiple measurements for %.2f seconds", highDrift, c.NtpMeasurementDuration.Seconds())
+		log.Printf("WARN: clock drift is above %.3fs, taking multiple measurements for %.2f seconds", c.NtpHighDrift.Seconds(), c.NtpMeasurementDuration.Seconds())
 		for time.Since(begin) < c.NtpMeasurementDuration {
 			nextMeasurement, err := c.getClockOffsetAndStratum()
 			if err != nil {
+				c.serverReachable.WithLabelValues(c.NtpServer).Set(0)
```

Comment:
Could we Set(0) at the top of measure() once?

Reply:
I'm not sure since there could be a disconnection during the time period depicted by `NtpMeasurementDuration` . However, we could perhaps set the value in the error condition of the function `getClockOffsetAndStratum`.